api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/argumentAxis/customPosition.md

```diff
@@ -10,4 +10,11 @@ Specifies the position of the argument axis on the value axis.
 ---
 The value of this option should be specified in the same format as the values on the value axis. The argument axis will stick to the pane border if the **customPosition**'s value is outside the value axis range.
```

Comment:
"The argument axis will stick to" What do you mean 'will stick to'?

Reply:
Let's say we have the following visual range on the value axis:  {-20; 20}. This means that the axis will have values only within this range. Then we specify the customPosition's value at 30. Since the pane (area on which the chart is drawn) end at 20, an axis cannot be located at this value. Instead, it will be located at the border of this pane.

Reply:
```suggestion
The value of this option should be specified in the same format as the values on the value axis. The argument axis is on the pane border if the **customPosition**'s value is outside the value axis range.
```